Movecraft/src/main/java/net/countercraft/movecraft/craft/BaseCraft.java

```diff
@@ -552,4 +559,9 @@ public void setTotalFuel(double fuel) {
     public double getTotalFuel() {
         return totalFuel;
     }
+
+    @Override
+    public CraftDataTagContainer getDataTagContainer() {
+        return dataTagContainer;
+    }
```

Comment:
Ye, we can remove that, .get() and .set() is enough

Reply:
Actually, java disallows private and protected modifiers in interfaces. Since that method is used by the interface default implementations of getDataTag() and setDataTag(), we also can't remove it

Reply:
Is there a reason we need to expose the `CraftDataTagContainer` at the craft level?  I think we could just leave it totally internal to `BaseCraft`.